docs/control-structures.rst

```diff
@@ -50,20 +55,50 @@ A Vyper contract cannot call directly between two external functions. If you mus
 Internal Functions
 ******************
 
-Internal functions (marked with the ``@internal`` decorator) are only accessible from other functions within the same contract. They are called via the :ref:`self<constants-self>` object:
+Internal functions (optionally marked with the ``@internal`` decorator) are only accessible from other functions within the same contract. They are invoked via the :ref:`self<constants-self>` object:
 
 .. code-block:: vyper
 
-    @internal
-    def _times_two(amount: uint256, two: uint256 = 2) -> uint256:
-        return amount * two
+    def _times_two(amount: uint256) -> uint256:
+        return amount * 2
 
     @external
     def calculate(amount: uint256) -> uint256:
         return self._times_two(amount)
 
+Or for internal functions which are defined in imported modules, they are invoked by prefixing the name of the module to the function name:
+
+.. code-block:: vyper
+    import calculator_library
+
+    @external
+    def calculate(amount: uint256) -> uint256:
+        return calculator_library._times_two(amount)
+
+.. note::
+   As of v0.4.0, the ``@internal`` decorator is optional. That is, functions with no visibility decorator default to being ``internal``.
+
 .. note::
-    Since calling an ``internal`` function is realized by jumping to its entry label, the internal function dispatcher ensures the correctness of the jumps. Please note that for ``internal`` functions which use more than one default parameter, Vyper versions ``>=0.3.8`` are strongly recommended due to the security advisory `GHSA-ph9x-4vc9-m39g <https://github.com/vyperlang/vyper/security/advisories/GHSA-ph9x-4vc9-m39g>`_.
+    Please note that for ``internal`` functions which use more than one default parameter, Vyper versions ``>=0.3.8`` are recommended due to the security advisory `GHSA-ph9x-4vc9-m39g <https://github.com/vyperlang/vyper/security/advisories/GHSA-ph9x-4vc9-m39g>`_.
+
+
+The ``__init__`` Function
+-------------------------
+
+.. _init-function:
+
+The ``__init__()`` function, also known as the constructor, is a special initialization function that is only called at the time of deploying a contract. It can be used to set initial values for storage or immutable variables. It must be declared with the ``@deploy`` decorator. A common use case is to set an ``owner`` variable with the creator the contract:
```

Comment:
```suggestion
The ``__init__()`` function, also known as the constructor, is a special initialization function that is only called at the time of deploying a contract. It can be used to set initial values for storage or immutable variables. It must be declared with the ``@deploy`` decorator. A common use case is to set an ``owner`` variable with the creator of the contract:
```
think there is a missing word or two here